data/Dockerfiles/dovecot/Dockerfile

```diff
@@ -101,13 +112,32 @@ RUN groupadd -g 5000 vmail \
   dovecot-core \
   dovecot-pop3d \
   dovecot-imapd \
-  dovecot-solr \
-  && pip3 install mysql-connector-python html2text jinja2 redis \
+  dovecot-dev
+  #dovecot-solr \
+RUN  cd /tmp && wget https://oligarchy.co.uk/xapian/1.4.19/xapian-core-1.4.19.tar.xz && tar xf xapian-core-1.4.19.tar.xz && cd xapian-core-1.4.19 \
+  && ./configure --prefix=/opt \
+  && make && sudo make install 
+
+RUN cd /tmp && git clone https://github.com/grosjo/fts-xapian && cd fts-xapian \
+  && autoupdate \
+  && autoreconf -vi \
+  && ./configure --with-dovecot=/usr/lib/dovecot \
+  && make && sudo make install
+
+RUN pip3 install mysql-connector-python html2text jinja2 redis \
   && apt-get autoremove --purge -y \
+  && apt-get purge -y \   
+     make \
+     libxapian-dev \
+     libicu-dev \
+     zlib1g-dev \
+     pkg-config \
+     build-essential \
   && apt-get autoclean \
   && rm -rf /var/lib/apt/lists/* \
-  && rm -rf /tmp/* /var/tmp/* /root/.cache/
+  && rm -rf /tmp/* /var/tmp/* /root/.cache/ 
```

Comment:
You need to merge everything into a single `RUN` command, otherwise this cleanup won‘t help shrink the image size.